lib/iiif/v3/presentation/canvas.rb

```diff
@@ -3,25 +3,22 @@ module V3
     module Presentation
       class Canvas < IIIF::V3::AbstractResource
 
-        # TODO (?) a simple 'Image Canvas' constructor.
-
-        TYPE = 'Canvas'
+        TYPE = 'Canvas'.freeze
 
         def required_keys
           super + %w{ id label }
         end
 
-        def array_only_keys
-          super + %w{ content }
+        def prohibited_keys
```

Comment:
Can you explain why you made the changes to remove int_only_keys and the additions to prohibited_keys and legal_viewing_hint_values?